pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2839,4 +2845,10 @@ public int getMetadataStoreOperationTimeoutSeconds() {
     public int getMetadataStoreCacheExpirySeconds() {
         return zooKeeperCacheExpirySeconds > 0 ? zooKeeperCacheExpirySeconds : metadataStoreCacheExpirySeconds;
     }
+
+    public long getManagedLedgerCacheEvictionIntervalMs() {
+        return managedLedgerCacheEvictionFrequency > 0
+                ? (long) (1000 / Math.max(Math.min(managedLedgerCacheEvictionFrequency, 1000.0), 0.001))
+                : Math.min(1000000, managedLedgerCacheEvictionIntervalMs);
```

Comment:
Where does this `1000000` comes from?

Reply:
the min value of managedLedgerCacheEvictionFrequency is  0.001 time per second, 
```
double evictionFrequency = Math.max(Math.min(config.getCacheEvictionFrequency(), 1000.0), 0.001);
```
So, it's 1000,000 if we transfer to cacheEvictionIntervalMs

Reply:
@Maybe making the `1000000` to be static and adding comments is better. 
Also, works with other numbers such as `1000` and `0.0.1`.